tests/nimony/nosystem/tfib2.nif

```diff
@@ -104,8 +104,7 @@
     (else 2,1
      (stmts 7
       (asgn ~7 result.0 11
-       (add ~6,~21
-        (i -1) ~6
+       (infix \2B ~6
```

Comment:
Looks wrong to me.

Reply:
Thought this at first too but this is actually the `+` in `fib(a-1) + fib(a-2)`, which should match to the concept proc which converts to the identifier. What I think happened before is: `-` in `a-1` and `a-2` matched to the concept proc and inferred `Self` to `int` and so the return type instantiated to `int`, then the `+` matched to the `int` overload. Now `Self` infers to `T` and stays that way.